hub/policy/src/integration-test/java/uk/gov/ida/integrationtest/hub/policy/apprule/AuthnRequestFromTransactionResourceIntegrationTest.java

```diff
@@ -189,6 +189,20 @@ public void shouldRestartEidasJourney() {
         assertThat(checkStateChanged.readEntity(String.class)).isEqualTo(SessionStartedState.class.getName());
     }
 
+    @Test
+    public void shouldRestartJourney() {
+        sessionId = SessionId.createNewSessionId();
+        TestSessionResourceHelper.createSessionInEidasAuthnFailedErrorState(sessionId, client, buildUriForTestSession(EIDAS_AUTHN_FAILED_STATE, sessionId));
```

Comment:
Is this the only state we need to test for? From the blurb I thought it was also the `IDPSelected` state.

Reply:
That's in part 2 of the Hub release which will enable `IdpSelected` -> `SessionStarted` transition